### DISCUSSION TITLE
Immediate vs Carryover Response and bootstrapped ROI Plots

### DISCUSSION THREAD

**Discussion by ilborobio** [facebookexperimental/Robyn#743] [General]
Hello there,

I've just updated the Robyn library and I have discovered that there are two additional plots in one-model-pager. But I cannot find any reference in the official documentation and no info about their meaning.

I can imagine their surface meaning. But, like with the other plots, is there any place where I can find explanations about this two new plots? Or can someone explain them here?

Thanks a lot!

Iván

**Comment by gufengzhou** [comment]
Sorry for the late reply

More precision in MMM-experiment calibration with Robyn from Meta ... https://medium.com/@gufengzhou/more-precision-in-mmm-experiment-calibration-with-robyn-from-meta-marketing-science-f608841fc6d4
